src/runtime/Global.h

```diff
@@ -20,6 +20,10 @@
 #ifndef __EscargotGlobal__
 #define __EscargotGlobal__
 
+#if defined(ENABLE_THREADING) && !defined(HAVE_BUILTIN_ATOMIC_FUNCTIONS)
```

Comment:
I suggest using new macro ```ENABLE_SPINLOCK``` mean this conditions

Reply:
The condition is not same with ENABLE_SPINLOCK

Reply:
But I added new define statement for clean code

Reply:
Now it's clearer!